rest/v10/poll.ts

```diff
@@ -0,0 +1,43 @@
+import type { Snowflake } from '../../globals';
+import type { APIMessage, APIPoll, APIUser } from '../../v10';
+
+/**
+ * https://discord.com/developers/docs/resources/poll#get-answer-voters
+ */
+export interface RESTGetAPIPollAnswerVotersQuery {
+	/**
+	 * Get users after this user ID
+	 */
+	after?: Snowflake;
+	/**
+	 * Max number of users to return (1-100)
+	 *
+	 * @default 25
+	 */
+	limit?: number;
+}
+
+/**
+ * https://discord.com/developers/docs/resources/poll#poll-create-request-object-poll-create-request-object-structure
+ */
+export interface RESTAPIPollCreate extends Omit<APIPoll, 'answer_id' | 'expiry' | 'results'> {
```

Comment:
`answer_id` is not right here. You have to change the `answers` property instead